clusterloader2/docs/experiments.md

```diff
@@ -0,0 +1,97 @@
+# Clusterloader2 - experiment rollout
+
+In this doc any change to the behavior of clusterloader2 is referred as an
+"experiment".
```

Comment:
Maybe it's actually nit-picking, but I'm wondering what "change to behavior" really means.
It's very vague and I would like to avoid a pathological case when we will be gating every single change to CL2.

I think we should be more explicit and talk about:
- new measurements?
- semantic changes to existing measurements?
- something else?

@mm4tt for thoughts

Reply:
I left it deliberately vague, so if you wonder whether your change is actually changing behavior or not, you should default to enable it gradually. However I am happy to be more precise here. 

I agree with your list. I'd add to the list major changes to clusterloader that alters how test are setup and run (such us enabling Prometheus or enabling node exporter).

Reply:
+1 to being more precise and including things that we're 100% sure that should be gated. I believe for the rest, we (and others) should just use common sense.

Reply:
Given that there is a very limited set of people who can approve changes to it, I would start with what there is (I'm fine what you suggested in addition to my examples) and we can extend this list when we better-defined things have a potential to break stuff.

Reply:
I'm sorry - by "what there is", I though about my list with your addition, not with there is in the doc now :)